build.gradle.kts

```diff
@@ -81,6 +81,21 @@ tasks {
         dependsOn("generateBundle")
         dependsOn("generateMeta")
     }
+    register<JavaExec>("generateMergedStrings") {
+        description = "Generate a merged English strings.xml file, used for crowdin translations"
```

Comment:
> Because the task only merges the individual patch string files, which are English.

I believe merging is not necessary. Crowdin can accept a wildcard for source files to translate.

If the only reason for merging was to support Crowdin this can safely be removed.